src/app/routes/getInitialData/index.test.js

```diff
@@ -62,21 +62,23 @@ describe('getInitialData', () => {
     });
   });
 
-  describe('On client', () => {
+  describe('Using www.test data', () => {
```

Comment:
The previous client test is redundant and the change to pointing at test/local data isn't actually returning any data. API is [mocked](https://github.com/BBC-News/simorgh/blob/7cf4c5cff9daeb629034ad48d77df4e9ae6b0432/src/app/routes/getInitialData/index.test.js#L16-L31) and it doesn't actually hit any endpoint so it doesn't matter what the URL is.

As the logic is now the same on the server and client, there can just be a single block of tests for testing the interaction with the fetch API.